src/pages/using-spark/components/promo.mdx

```diff
@@ -0,0 +1,106 @@
+---
+  title: Promo
+---
+import ComponentPreview from '../../../components/ComponentPreview';
+import { SprkDivider } from '@sparkdesignsystem/spark-react';
+
+# Promo
+
+Promo is a brief message and call to action
```

Comment:
Should call to action be call-to-action or call to action?

Reply:
Just did a quick internet search and see it both ways. 🤷‍♀️